creator-node/src/apiHelpers.ts

```diff
@@ -1,21 +1,64 @@
-const config = require('./config')
+import config from './config'
+import { Request, Response } from 'express'
 
-const {
+import {
   requestNotExcludedFromLogging,
   getDuration,
   setFieldsInChildLogger
-} = require('./logging')
-const { generateTimestampAndSignature } = require('./apiSigning')
+} from './logging'
+import { generateTimestampAndSignature } from './apiSigning'
 
-module.exports.handleResponse = (func) => {
-  return async function (req, res, next) {
-    try {
-      const resp = await func(req, res, next)
 
-      if (!isValidResponse(resp)) {
-        throw new Error('Invalid response returned by function')
-      }
+// constants
+const HEARTBEAT_INTERVAL: number = 5000
+
+const StatusCode = {
+  OK: 200,
+  BAD_REQUEST: 400,
+  UNAUTHORIZED: 401,
+  FORBIDDEN: 403,
+  NOT_FOUND: 404,
+  REQUESTED_RANGE_NOT_SATISFIABLE: 416,
+  INTERNAL_SERVER_ERROR: 500
+} as const
```

Comment:
I'm not opposed to this but what about pulling in this package: https://github.com/prettymuchbryce/http-status-codes

It has them all defined with reason phrases and 0 dependencies